shared/actions/wallets.js

```diff
@@ -98,6 +98,7 @@ const openSendRequestForm = (state: TypedState, action: WalletsGen.OpenSendReque
           WalletsGen.createSetBuildingCurrency({
             currency:
               action.payload.currency ||
+              (state.wallets.lastSentXLM && 'XLM') ||
```

Comment:
=== ?

Reply:
thanks!

Reply:
whoops actually I thought that this was a different file. not a typo! if there isn't an `action.payload.currency` and we know `lastSentXLM`, this sets the currency to `XLM`.

Reply:
k